### ISSUE TITLE
type definition for  runner Video options is missing

### ISSUE THREAD

**Issue by devmondo** [DevExpress/testcafe#3870]
### What is the Current behavior?
using vscode or webstorm we cant see the video options in the auto complete pop up. going through `index.d.ts` you would see that the video options typing is not there too.
 
### What is the Expected behavior?
to be able to autocomplete video options based on the type definition file.
 
### screenshot
![image](https://user-images.githubusercontent.com/1266367/59029548-98ee1d00-886f-11e9-9088-8dedb580d5d4.png)

### Your Environment details:
 
* testcafe version:                   1.2.0
* node.js version:                    12
* platform and version:          Windows 10

**Comment by Favna** [comment]
Ran into the same issue so for other readers, the temporary workaround of using module augmentation will work:

```ts
declare global {
  interface VideoOptions {
    singleFile?: boolean;
    failedOnly?: boolean;
    pathPattern?: string;
  }

  interface Runner {
    video: (path: string, options?: VideoOptions, encodingOptions?: unknown) => this;
  }
}
```

I didn't know what type to properly assign to encodingOptions as it supports "all of ffmpeg options" so I left it as `unknown`.

**Comment by AndreyBelym** [comment]
Closed by #4636.
